### PR TITLE
VSL-56: QSP-3 Insufficient Testing

### PR DESCRIPTION
**Pull request by jacksonConrad**

### CODE REVIEW

**Comment by KsenijaZivkovic** [comment]
Added commits after PR review and increased coverage for destroyAction, and added edge case for signerRevokeApproval. Also, removed the unused test.cdc

**Review by KsenijaZivkovic — APPROVED** [review]

File: contracts/DAOTreasury.cdc [hunk 1]
```diff
@@ -62,6 +64,12 @@ pub contract DAOTreasury {
       emit ExecuteAction(actionUUID: actionUUID)
     }
 
+    pub fun destroyAction(actionUUID: UInt64) {
```
Comment (jacksonConrad) [inline contracts/DAOTreasury.cdc:67]: `DestroyAction` should be its own MultiSignAction in `TreasuryActions` and require signers to approve it before executing like other multi-sign actions.  Exposing via `TreasuryPublic` interface would allow anyone to delete a proposed action :)

Comment (KsenijaZivkovic) [reply contracts/DAOTreasury.cdc:67]: On it!
